examples/ui_elements/forms/text_input.xml

```diff
@@ -242,6 +243,31 @@
             value=""
           />
         </view>
+        
+        <view style="FormGroup">
+          <text style="label">Noneditable Text</text>
+          <text-field
+            placeholderTextColor="#8D9494"
+            mask="999 99 9999"
```

Comment:
Please remove the `mask` attribute from this example and the one below, since they are not relevant to the `editable` feature.